src/main/java/com/PBA/budgetservice/controller/IncomeController.java

```diff
@@ -39,4 +39,11 @@ public ResponseEntity<IncomeDto> createIncome(
     public ResponseEntity<IncomeDto> updateIncome(
             @io.swagger.v3.oas.annotations.parameters.RequestBody(description = "Income to update")
             @RequestBody IncomeUpdateRequest incomeUpdateRequest, @PathVariable("uid") UUID uid);
+
+    @Operation(summary = "Deletes the income with the given uid from the system, if it exists.")
+    @ApiResponses(value = {
+            @ApiResponse(responseCode = "200", description = "OK")
+    })
```

Comment:
You are missing status code 404, not found, or 400 bad request if parameter is invalid